internal/resolution/variable_sources/olm/olm_test.go

```diff
@@ -177,8 +177,8 @@ var _ input.EntitySource = &FailEntitySource{}
 type FailEntitySource struct {
 }
 
-func (f FailEntitySource) Get(ctx context.Context, id deppy.Identifier) *input.Entity {
-	return nil
+func (f FailEntitySource) Get(ctx context.Context, id deppy.Identifier) (*input.Entity, error) {
+	return nil, fmt.Errorf("error executing get")
```

Comment:
Not a blocker at all: but do we want to define this as constants?
panicEntitySrcErr, FailEntitySrcErr etc.. just in case we want to use it in tests to assert the errors.

Reply:
I see how it might be useful, but I would like to avoid doing it in this PR. Focus of this is to upgrade Deppy with minial changes to operator controller.

This function follows the existing pattern for `Filter`, `GroupBy`, etc.